python/pyspark/sql/readwriter.py

```diff
@@ -388,16 +388,18 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
         return self._df(self._jreader.csv(self._spark._sc._jvm.PythonUtils.toSeq(path)))
 
     @since(1.5)
-    def orc(self, path):
-        """Loads an ORC file, returning the result as a :class:`DataFrame`.
+    def orc(self, paths):
```

Comment:
So if someones been calling orc with a named param of path this could cause them problems when they upgrade. I might be being overly cautious but it seems like we should avoid breaking that since we don't have to until the next major version change.